test/Microsoft.AspNet.Mvc.DataAnnotations.Test/DataAnnotationsClientModelValidatorProviderTest.cs

```diff
@@ -127,15 +128,8 @@ public static IEnumerable<object[]> DataAnnotationAdapters
             ValidationAttribute attribute,
             Type expectedAdapterType)
         {
-            // Arrange
-            var adapters = new DataAnnotationsClientModelValidatorProvider(
-                new TestOptionsManager<MvcDataAnnotationsLocalizationOptions>(),
-                stringLocalizerFactory: null)
-                .AttributeFactories;
-            var adapterFactory = adapters.Single(kvp => kvp.Key == attribute.GetType()).Value;
-
-            // Act
-            var adapter = adapterFactory(attribute, stringLocalizer: null);
+            // Arrange and Act
+            var adapter = ValidationAttributeAdapterProvider.GetAttributeAdapter(attribute, null);
```

Comment:
If these are now tests of `ValidationAttributeAdapterProvider`, move them to `ValidationAttributeAdapterProviderTest`.